src/client/websocket/WebSocketShardManager.js

```diff
@@ -35,14 +35,16 @@ class WebSocketShardManager extends EventEmitter {
 
   _spawnAll() {
     this.client.emit('debug', `Spawning ${this.shardCount} shard(s)`);
-    this.spawn(0);
-    const interval = setInterval(() => {
-      if (this.managers.size >= this.shardCount) {
-        clearInterval(interval);
-        return;
-      }
-      this.spawn(this.managers.size);
-    }, 5500);
+    const spawnLoop = (id) => {
+      const manager = this.spawn(id);
+      manager.on('ready', () => {
```

Comment:
should be `.once(`, also try and have a fallback for failed connections to just continue with the next shard